submissions/TUstiugov/html-css-popup/index.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="shortcut icon" href="./images/popup-icons/favicon.png" type="image/x-icon">
+  <link rel="stylesheet" href="./style.css">
+  <title>HTML-CSS-Popup</title>
+</head>
+
+<body>
+  <header class="header">
+    <nav class="header__navigation navigation">
+      <ul class="navigation__list flex-container">
+        <li class="navigation__item">
+          <a href="#" class="navigation__link navigation__link_mail">Mail</a>
+        </li>
+        <li class="navigation__item">
+          <a href="#" class="navigation__link navigation__link_images">Images</a>
+        </li>
+        <li class="navigation__item">
+          <a href="#popup" class="navigation__link navigation__link_popup"></a>
+          <div id="popup" class="popup">
+            <a href="#" class="popup__area"></a>
```

Comment:
A very cool solution, but in this case, you need to add `aria-label` attribute with a description: `Close modal` or something like that, also, it will be nice to display this element to indicate to the user where the focus is, but in your case, it's overlay behind the popup, so just add `aria-label` attribute with corresponding value)

Reply:
Done.